controllers/controller.js

```diff
@@ -50,4 +54,53 @@ const {
     
         next(err);
       })
+  };
+
+  exports.fetchArticlesComment = (req, res, next) => {
+
+    const { article_id } = req.params;
+
+    Promise.all([
+      selectArticleById(article_id),
+      selectArticleComment(article_id)
+    ])
+
+    .then(([articleExist ,comment]) => {
+      if(!articleExist){
```

Comment:
Do we need this error handler here? As we're invoking selectArticleById with our id that model should throw an error if that article doesn't exist based on previous logic. Try removing this and see if everything still works.

As a general rule we shouldn't have any logic implemented in the controller.